.github/workflows/test-aks.yaml

```diff
@@ -0,0 +1,363 @@
+    types: [opened, synchronize, reopened]
+    branches:
+      - v*.*
+      - edge
+  schedule: # Run every 2 hours
+    - cron: "0 */2 * * *"
+env:
+  RUN_IDENTIFIER: samplestest-${{ github.run_id }}-${{ github.run_attempt }}
+jobs:
+  test:
+    name: Sample tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: demo
+            os: ubuntu-latest
+            runOnPullRequest: true
+            app: demo
+            env: default
+            path: ./samples/demo/app.bicep
+            deployArgs: --application demo -p image=ghcr.io/radius-project/samples/demo:${{ github.run_id }}
+            exposeArgs: --application demo
+            uiTestFile: tests/demo/demo.app.spec.ts
+            port: 3000
+            container: demo
+            enableDapr: false
+            images: samples/demo
+            directories: samples/demo/
+          - name: dapr
+            os: ubuntu-latest-m
+            runOnPullRequest: true
+            app: dapr
+            env: default
+            path: ./samples/dapr/dapr.bicep
+            deployArgs: -p frontendImage=ghcr.io/radius-project/samples/dapr-frontend:${{ github.run_id }} -p backendImage=ghcr.io/radius-project/samples/dapr-backend:${{ github.run_id }}
+            enableDapr: true
+            images: samples/dapr-frontend,samples/dapr-backend
+            directories: samples/dapr/ui/,samples/dapr/nodeapp/
+          - name: volumes
+            os: ubuntu-latest
+            runOnPullRequest: true
+            app: myapp
+            env: default
+            path: ./samples/volumes/app.bicep
+            deployArgs: -p image=ghcr.io/radius-project/samples/volumes:${{ github.run_id }}
+            enableDapr: false
+            images: samples/volumes
+            directories: samples/volumes/
+          - name: eshop-containers
+            os: ubuntu-latest-m
+            runOnPullRequest: true
+            app: eshop
+            env: default
+            path: ./samples/eshop/eshop.bicep
+            uiTestFile: tests/eshop/eshop.app.spec.ts
+            enableDapr: false
+          - name: eshop-azure
+            os: ubuntu-latest-m
+            runOnPullRequest: true
+            app: eshop
+            env: azure
+            path: ./samples/eshop/eshop.bicep
+            uiTestFile: tests/eshop/eshop.app.spec.ts
+            credential: azure
+            enableDapr: false
+          - name: eshop-aws
+            os: ubuntu-latest-m
+            runOnPullRequest: true
+            app: eshop
+            env: aws
+            path: ./samples/eshop/eshop.bicep
+            uiTestFile: tests/eshop/eshop.app.spec.ts
+            credential: aws
+            enableDapr: false
+    env:
+      BRANCH: ${{ github.base_ref || github.ref_name }}
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      AZURE_LOCATION: westus3
+      AKS_RESOURCE_GROUP: samples-functional-tests
+      AKS_CLUSTER_NAME: samples-aks
```

Comment:
@ytimocin If you use the existing aks cluster, workflow must clean up the existing resource first before starting the test. I recommend to add the code to delete namespace first before starting the test.